independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json

```diff
@@ -0,0 +1,120 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Google Chart QR Code Generator",
+    "description": "This Connector is build arount the Google Chart API to generate QR codes to be used in Power Platform applications",
```

Comment:
@Ionavoss please remove the word "API" from the description. please follow our requirements [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

@Ionavoss All files should be free of grammatical and spelling errors. in the description you have "This Connector is build arount the Google Chart...", maybe you meant to say "This Connector is **built around** the Google Chart..."? Can you please fix?